backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlRepository.java

```diff
@@ -89,26 +96,51 @@ public static String toTrlTableName(final String tableName)
 		return tableName + TRL_TABLE_SUFFIX;
 	}
 
-	public final POTrlInfo createPOTrlInfo(final String tableName, @Nullable final String keyColumnName, final List<String> translatedColumnNames)
+	public static String toBaseTableName(final String tableName)
 	{
-		if (keyColumnName == null || Check.isEmpty(keyColumnName))
+		if(!tableName.endsWith(TRL_TABLE_SUFFIX))
 		{
-			return POTrlInfo.NOT_TRANSLATED;
+			return "";
 		}
+		final String baseTableName = StringUtils.replace(tableName, TRL_TABLE_SUFFIX, "");
```

Comment:
pls substring... blindly replacing might fail for cases like `Some_TrlWorld_Trl`... yup, pretty corner case, but being precise does not hurt.

Reply:
still pending